app/src/main/java/com/internship/move/ui/authentication/register/DrivingLicenseFragment.kt

```diff
@@ -0,0 +1,67 @@
+package com.internship.move.ui.authentication.register
+
+import android.net.Uri
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.core.content.FileProvider
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.lifecycleScope
+import androidx.navigation.fragment.findNavController
+import com.internship.move.BuildConfig
+import com.internship.move.R
+import com.internship.move.databinding.FragmentDrivingLicenseBinding
+import com.zhuinden.fragmentviewbindingdelegatekt.viewBinding
+import java.io.File
+
+class DrivingLicenseFragment : Fragment(R.layout.fragment_driving_license) {
+
+    private val binding by viewBinding(FragmentDrivingLicenseBinding::bind)
+    private val takeImageResult = registerForActivityResult(ActivityResultContracts.TakePicture()) { isSuccess ->
+        if (isSuccess) {
+            latestTmpUri?.let { uri ->
```

Comment:
You could perform the `null` check together with `isSuccess`, since a null uri is also a "failed" state that needs an informational toast shown to the user